### PR TITLE
Change the value of the indicators to numbers in the report

### PR DESCRIPTION
**Pull request by jaimeferreira11**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

Put an `x` in the boxes that apply.

* [x] Lint and unit tests pass locally with my changes
* [ ] I have added necessary documentation (if appropriate)
* [ ] I have added tests that prove my fix is effective or that my feature works (if appropiate)

### Description

Please provide a short description of your pull request.

### CODE REVIEW

File: src/main/java/py/org/fundacionparaguaya/pspserver/reports/services/impl/SnapshotReportManagerImpl.java [hunk 1]
```diff
@@ -42,11 +42,19 @@
...
+            "Family Code", "Family Name", "Family Status",
+            "Snapshot Created At");
 
     private static final String CSV_DELIMITER = ",";
 
+    private static final String INDICATOR_RED = "RED";
+
+    private static final String INDICATOR_YELOW = "YELLOW";
+
+    private static final String INDICATOR_GREEN = "GREEN";
```
[the remark this answers is not in the document]
Comment (jaimeferreira11) [reply src/main/java/py/org/fundacionparaguaya/pspserver/reports/services/impl/SnapshotReportManagerImpl.java]: Cambios realizados. @rodrigojv

File: src/main/java/py/org/fundacionparaguaya/pspserver/reports/services/impl/SnapshotReportManagerImpl.java [hunk 2]
```diff
@@ -235,14 +278,27 @@ public String generateCSVSnapshotByOrganizationAndCreatedDate(SnapshotFilterDTO
...
-            toRet = toRet
-                    + (row.stream().map(Object::toString).collect(Collectors.joining(CSV_DELIMITER))).concat("\n");
+            toRet = toRet + (row.stream().map(Object::toString)
+                    .collect(Collectors.joining(CSV_DELIMITER))).concat("\n");
         }
 
         return toRet;
     }
+
+    private String getIndicatorValues(String value) {
```
[the remark this answers is not in the document]
Comment (jaimeferreira11) [reply src/main/java/py/org/fundacionparaguaya/pspserver/reports/services/impl/SnapshotReportManagerImpl.java]: Cambios realizados. @rodrigojv

[the remark this answers is not in the document]
Comment (rodrigojv) [reply src/main/java/py/org/fundacionparaguaya/pspserver/reports/services/impl/SnapshotReportManagerImpl.java]: Favor eliminar las constantes `private static INDICATOR_GREEN`, etc.

**Comment by jaimeferreira11** [comment]
Cambios realizados! @rodrigojv

**Pull request merged**